fleet/lib/ccs-grafana/values.yaml

```diff
@@ -51,9 +51,18 @@ datasources:
     datasources:
       - name: InfluxDB
         type: influxdb
-        url: http://influxdb.ccs-monitor:8086
         access: proxy
+        url: http://influxdb.ccs-monitor:8086
         isDefault: true
         database: grafana
         user: ${`${INFLUXDB_USERNAME}`}
-        password: ${`${INFLUXDB_PASSWORD}`}
+        jsonData:
+          httpMode: GET
+        secureJsonData:
+          password: ${`${INFLUXDB_PASSWORD}`}
+
+sidecar:
+  dashboards:
+    enabled: true
+    namespace: ~
```

Comment:
Does this result in dashboards in all namespaces being used?  That is not desirable on at least pillan.

Reply:
do'h that was for testing, going to change it back to its own namespace, but I think the default is like that and this can be removed.